apps/style/code-studio/code-studio.scss

```diff
@@ -0,0 +1 @@
+@import 'selectize/dist/css/selectize';
```

Comment:
If we intend `code-studio.scss` to act as a replacement for application.css in apps-land, it might be worth a descriptive comment to that effect here - and a warning in application.scss that moving forward it'd be better to add dependencies here than there.